### ISSUE TITLE
Add base of "about" components

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#127]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

Similarly to #119 we need to create the `aboutCard.vue` base of the about components that will then be changed based on whether they're for organizations or events. The component should have card-style applied to it. There should further be some props in the `<script>`:

```ts
name?: string;
description: string;
aboutType: string;  // what we pass `organization`, `event`, etc to
```

We'll add more props as they're needed for the variants :)

### Contribution

Happy to work on this or support someone who has interest. This issue is again just the base shell, with further issues following to add functionality for if it's for events or organizations.

**Comment by connorlunsford** [comment]
@andrewtavis Happy to work on this if you want. Should have time to do it by this weekend!

**Comment by andrewtavis** [comment]
Hey @connorlunsford! Great to hear from you again! :) :) I should have done a better job of defining the time scope of this one. I’ll likely do this this evening as we’ll be working on parts that depend on it tomorrow night at the code night I mentioned on Matrix 😊 Let’s plan on another issue, and you’d of course be welcome to participate tomorrow if you’re interested!

**Comment by connorlunsford** [comment]
@andrewtavis Haha apologies got a new phone and it looks like it signed me out of the app I was using to access matrix on it. Looks like (if I have my time zones correct) I'm going to have a conflict during the meetup so if you are doing a video call or anything I won't be able to join. I'll keep updated on matrix and I'll keep an eye on the issues to see if any would be good to assign to me. Thanks!

**Comment by andrewtavis** [comment]
Looking forward to the next issue, @connorlunsford! 😊

**Comment by andrewtavis** [comment]
Closed in f57b1e8 😊
